Modules/Loadable/Markups/Testing/Python/MarkupsCurveMeasurementsTest.py

```diff
@@ -44,15 +44,57 @@
 
 curvatureArray = curvePointData.GetArray(1)
 if curvatureArray.GetMaxId() != curvePointData.GetNumberOfTuples()-1:
-  exceptionMessage = "Unexpected number of values in curvature data array: " + str(curvatureArray.GetMaxId())
+  exceptionMessage = "Unexpected number of values in curvature data array: %d (expected %d)" % (curvatureArray.GetMaxId(), curvePointData.GetNumberOfTuples()-1)
   raise Exception(exceptionMessage)
 
-if abs(curvatureArray.GetMaxNorm() - 0.9816015970208652) > 0.0001:
-  exceptionMessage = "Unexpected maximum norm in curvature data array: " + str(curvatureArray.GetMaxNorm())
+if abs(curvatureArray.GetRange()[0] - 0.0) > 0.0001:
+  exceptionMessage = "Unexpected minimum in curvature data array: " + str(curvatureArray.GetRange()[0])
+  raise Exception(exceptionMessage)
+if abs(curvatureArray.GetRange()[1] - 0.9816015970208652) > 0.0001:
+  exceptionMessage = "Unexpected maximum in curvature data array: " + str(curvatureArray.GetRange()[1])
   raise Exception(exceptionMessage)
 
 # Turn off curvature computation
 curveNode.SetCalculateCurvature(False)
 if curvePointData.GetNumberOfArrays() != 1:
   exceptionMessage = "Unexpected number of data arrays in curve: " + str(curvePointData.GetNumberOfArrays())
   raise Exception(exceptionMessage)
+
+print('Open curve curvature test finished successfully')
+
+#
+# Test closed curve curvature computation
+#
+closedCurveNode = slicer.mrmlScene.AddNewNodeByClass('vtkMRMLMarkupsClosedCurveNode')
+pos = [0]*3
+for i in range(curveNode.GetNumberOfControlPoints()):
+  curveNode.GetNthControlPointPosition(i, pos)
+  posVector = vtk.vtkVector3d()
+  posVector.Set(pos[0], pos[1], pos[2])
+  closedCurveNode.AddControlPoint(posVector)
```

Comment:
you can pass the vector in the vtkVector3d constructor: `closedCurveNode.AddControlPoint(vtk.vtkVector3d(pos))`